packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -784,7 +784,7 @@ class _RenderScrollSemantics extends RenderProxyBox {
     _innerNode ??= SemanticsNode(showOnScreen: showOnScreen);
     _innerNode
       ..isMergedIntoParent = node.isPartOfNodeMerging
-      ..rect = Offset.zero & node.rect.size;
+      ..rect = node.rect;
```

Comment:
This code looks correct to me, I am curious though why we want to & with the offset.zero at the first place @goderbauer

Reply:
I think we forgot to change this when the code was moved in https://github.com/flutter/flutter/pull/13983 a loong time ago!